### ISSUE TITLE
Clarify hidden nodes with child nodes, referenced by aria-labelledby 

### ISSUE THREAD

**Issue by jnurthen** [w3c/accname#48]
The spec does not make any allowance for hidden child nodes referenced with aria-labelledby. 
`
<input aria-labelledby="a">
<div id="a" hidden>
   <span>Text</span>
</div>
`

Reading the algorithm the input should have no accessible name as the span is not directly references by aria-labelledby but is hidden.

When testing Chrome and Firefox they both return an accessible name of Text.

We should fix the algorithm to match the browser implementations.

**Comment by jnurthen** [comment]
@dylanb - logged the issue for you. Please clarify if this is not correct.

**Comment by dylanb** [comment]
Correct but more work is necessary to ensure the algorithm produces the desired result in all edge cases (including inputs with and without values)
